homeassistant/helpers/enum.py

```diff
@@ -0,0 +1,20 @@
+"""Enum related helpers."""
```

Comment:
Consider moving this to util. We normally distinguish helpers and util as: helpers are tools that are aware of Home Assistant while utils are not aware.

Reply:
```suggestion
"""Enum related helpers."""
from __future__ import annotations
```